Uchu.Core/Serializable/Struct/Attribute/PacketStructAttribute.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using RakDotNet;
+
+namespace Uchu.Core
+{
+    /// <summary>
+    /// Indicates a bit flag is used for a property.
```

Comment:
It is. If a property has a `Default` (except for `bool`s), it will write a bit flag and the value for non-default values, and just a bit flag for default values.

Reply:
That makes sense, but I mean that this description is used for [DefaultAttribute](https://github.com/UchuServer/Uchu/blob/5cd7dfd77aa541c5a59be7da9ef8d1bc45d7f89c/Uchu.Core/Serializable/Struct/Attribute/DefaultAttribute.cs), it seems unrelated to the `PacketStructAttribute` defined in this file.

Reply:
Didn't realize the file name was that. I'll correct that.